src/NServiceBus.Transport.SQS.AcceptanceTests/ConfigureEndpointSqsTransport.cs

```diff
@@ -49,7 +49,7 @@ static void ApplyMappingsToSupportMultipleInheritance(string endpointName, Trans
 
             if (endpointName == Conventions.EndpointNamingConvention(typeof(When_started_by_base_event_from_other_saga.SagaThatIsStartedByABaseEvent)))
             {
-                transportConfig.MapEvent<When_started_by_base_event_from_other_saga.BaseEvent, When_started_by_base_event_from_other_saga.SomethingHappenedEvent>();
+                transportConfig.MapEvent<When_started_by_base_event_from_other_saga.IBaseEvent, When_started_by_base_event_from_other_saga.ISomethingHappenedEvent>();
```

Comment:
Was this needed for .NET 5?

Reply:
Wasn't compiling otherwise

Reply:
Is this something we need to look into?

Reply:
Here's the change: **https://github.com/Particular/NServiceBus/commit/a0544a8dd9471f2894dc42af5b10d056c60198de#diff-5199b4b4c7ce13e7ff0200a6979bdaba30661c4af4ae1076aebf57fdb60e8607**

Reply:
I think this showed up here and not in the others so far is because of the explicit mapping required.